Terminal.Gui/Views/Shortcut.cs

```diff
@@ -461,18 +498,34 @@ public View CommandView
             _commandView.CanFocus = false;
 
             _commandView.HotKeyChanged += (s, e) =>
-                                          {
-                                              if (e.NewKey != Key.Empty)
-                                              {
-                                                  // Add it 
-                                                  AddKeyBindingsForHotKey (e.OldKey, e.NewKey);
-                                              }
-                                          };
+            {
+                if (e.NewKey != Key.Empty)
+                {
+                    // Add it 
+                    AddKeyBindingsForHotKey (e.OldKey, e.NewKey);
+                }
+            };
 
             _commandView.HotKeySpecifier = new ('_');
 
             Title = _commandView.Text;
 
+            _commandView.Accept += CommandViewOnAccept;
+
+            void CommandViewOnAccept (object sender, HandledEventArgs e)
+            {
+                KeyBinding binding = new ([Command.Select], KeyBindingScope.Focused, _commandView, null);
+                e.Handled = DispatchAcceptCommand (new CommandContext (Command.Select, null, binding)) == true;
+            }
```

Comment:
Could use some code comments clarifying what is going on

Reply:
This part of Shortcut is still a WIP. 

This PR is prep for #3691 and replacing `Menu` with `Bar`. So don't worry about it ;-)._